File: division.py
x=int(input("Enter x:"))
y=int(input("Enter y:"))
if(y!=0):
     if((type(x)==int and type(y)==int) or (type(x)==float and type(y)==float) or(type(x)==float and type(y)==int) or (type(x)==int and    type(y)==float)):
         print(x/y)
else:
    print("division not possible")

File: division.m
x = input("Enter x: ");
y = input("Enter y: ");
if (y != 0)
    if (isa(x, 'double') && isa(y, 'double')) || (isa(x, 'float') && isa(y, 'float')) || (isa(x, 'float') && isa(y, 'double')) || (isa(x, 'double') && isa(y, 'float'))
        disp(x/y);
    end
else
    disp("Division not possible");
end